Model/Order/CompleteOrderPool.php

```diff
@@ -67,14 +68,21 @@ public function execute(OrderInterface $order): void
             QuoteExtensionDataResource::QUOTE_ID
         );
         $flowType = $quoteExtensionData->getApiType();
-        $processor = $this->pool[$flowType] ?? null;
-        if (!($processor instanceof CompleteOrderInterface)) {
-            $this->logger->error(
-                __('Failed to find complete processor for order with id="%1"', $order->getEntityId())
-            );
 
-            return;
+        // TODO: Remove logic around order pool as we only need to run logic here for simple order types
+        // For 'default' order types all order complete processing happens in the platform connector
+        // TODO: Update naming around simple/non-simple orders to signal it relates to source of truth instead of order type
+        if ($flowType === self::API_TYPE_SIMPLE) {
```

Comment:
I think @CalinR and @bobperkinson mention of us making this actually a `!== 'default'`.

Reply:
Correct, this should be !== default as everything else needs to go through the processor

Reply:
Oh yes you're right, updated to !== default